### DISCUSSION TITLE
Dates wrong every 2048th row

### DISCUSSION THREAD

**Discussion by slvr611** [dbeaver/dbeaver#22301] [Q&A] [answered]
I am programming in python using DuckDB to interface with a sql database and DBeaver to view the database.  I'm getting a lot of input data that is entered into my database in batches of around 1.8k rows at a time.  Every 2048th row, starting at row 2049 is having a problem where when viewing the row in a full table view it displays everything correctly, but when querying for the row with that specific ID it has the wrong date and sometimes another integer on the row is wrong as well, sometimes negative when it should only ever be positive, although this is not consistent.  To me that indicates something overflow related.  This wrong date persists into queries from my main python script using DuckDB, but the really strange thing to me is that it displays the correct date in the full table view, but when queried for it displays the incorrect date.

My inclination is that the problem is related to the underlying database, as it's every 2048th row regardless of the size the batches are inserted in, and I suspect the problem is DuckDB related because it occurs directly on insertion into the database, although it could be some property of the SQL database itself that is messing with things rather than DuckDB.  With that being said, I figured I should cast a wider net and there is also clearly something wonky going on with the way it is displayed in DBeaver

Any input and advice is greatly appreciated, and I will gladly share additional info as needed, although some of it may need to be via DM.

**Comment by ShadelessFox** [comment]
Hello @slvr611,

What driver version are you using? Have you tried using the latest version? Also, footage showing the issue could be helpful.

**Reply by slvr611** [reply]
Hiya @ShadelessFox,

After making the included video of the problem I updated to the newest version of the driver, which matches the version of DuckDB, v0.9.2  The problem still ocurred on that new version of the driver

https://youtu.be/yT9NRyOJAsA

As you can see in this video, in the data view of the table the dates appear correct, but on querying for that specific line, the dates are no longer correct

**Reply by ShadelessFox** [reply, marked as answer]
Thanks for the provided information. We don't solve bugs posted in discussions, so can you please create a new issue instead?

**Reply by slvr611** [reply]
Thanks, I'll do that shortly
